LICENSE-BSD-Chronium

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
```

Comment:
Typo in file name. Probably we want to say "Chromium".

Reply:
Yes, will rename this file